sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_stats_clients_frst_seen_v2/query.sql

```diff
@@ -0,0 +1,167 @@
+-- Get all clients who had activity in the last 28 days and their bit patterns.
+WITH submission_date_activity AS (
```

Comment:
We should change the naming of this cte to show that it's pulling active clients. Something like `active_clients` or `client_activity`. Don't think the submission_date is giving me any additional info about what's being pulled in the CTE.

Reply:
or we could even be more literal `client_main_ping_activity` since as an org on desktop we now refer to being "Active" as having > 0 active hours and URIs.

Reply:
replaced submission_date_activity with client_activity